opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -42,4 +42,18 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successfull export request.
+  // When the request is only partially successfull (due to e.g. invalid/bad data),
+  // the server MUST populate the details property with a developer-facing error
+  // message along with the number of accepted log records.
+  ExportLogsPartialSuccess details = 1;
+}
+
+message ExportLogsPartialSuccess {
+  // The number of accepted log records
+  int64 accepted_log_records = 1;
```

Comment:
If we invert this to be `rejected_log_records` it would be 0 under normal circumstances. I'm not a protobuf expert, but I imagine that could result in a small payload most of the time.

Reply:
I vote for accepted instead of rejected.

Reply:
I also like "accepted" a bit more since it explicitly can be compared with the number of the records sent.
If all records are accepted then I think the expectation is that the entire ExportLogsPartialSuccess os omitted so the payload size issue is irrelevant.

Reply:
I will resolve this as `accepted` is what we are leaning towards. And like @tigrannajaryan mentioned, in case a export is successful (all logs, spans or metrics) are ingested this property will be null, so no change in the payload size compared to what we have now.